editor/edit-post/modes/keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
While I'd like for us to flatten and eliminate the `modes` directory altogether, I don't particularly like the creation of a new folder here, because as it was prior to these changes, the `modes` directory contains exclusively the supported editing modes for the editor: visual and text. The shortcuts are not in themselves a mode.